### DISCUSSION TITLE
form.isCaptchaEnabled always returns true

### DISCUSSION THREAD

**Discussion by wolispace** [solspace/craft-freeform#1317] [Feature Requests]
### What happened?

Added an integration for reCAPTCHA and we want to conditionally display the "This site is protected by reCAPTCHA..." text based on whether the integration is enabled on the form.

There isn't any official documentation on how to check if captchas are enabled or not, but found this function that seems to be relavent.
https://github.com/solspace/craft-freeform/blob/ac06755a96d2f255c80d786856b723c259035bfa/packages/plugin/src/Form/Form.php#L289-L296

Unfortunately this code will always show because the value is constantly true
```twig
{% if form.isCaptchaEnabled %}
...
{% endif %}
```

I have tried toggling the integration within the form and even deleting all captcha integration in the freeform settings but nothing seems to change this value.

![image](https://github.com/solspace/craft-freeform/assets/2208932/a33dafda-d6b9-419e-8683-3ab342c7deeb)

![image](https://github.com/solspace/craft-freeform/assets/2208932/8f241751-6751-47aa-802b-0d1cc5a8472e)

### How can we reproduce this?

1. Create a new form
2. in the form template add `{% dd form.isCaptchaEnabled %}`
3. try toggling captcha on/off in the form

### Freeform Edition

Pro

### Freeform Version

5.1.12

### Craft Version

Craft Pro 4.9.4

### When did this issue start?

Unsure

**Comment by kjmartens** [comment]
Hi @wolispace,

Unfortunately I don't believe there is a way to check this. I will confirm with a developer, but in the meantime, I have noted this as a feature request. 🙂

**Reply by rellafella** [reply]
@kjmartens I am not sure that I would agree with this being a feature request. The function exists to check whether the captcha is disabled, but it doesn't function in a predictable way, nor is it documented. I would consider this a bug.

**Reply by kjmartens** [reply]
I can see why this seems confusing, but this conditional is not designed for this use case. Unfortunately, nothing is currently available for the use case you describe, but we will consider adding something in the future. 🙂

**Comment by wolispace** [comment]
Yes, form.isCaptchaEnabled always returning true does seem like a bug to me.

**Reply by kjmartens** [reply]
I can see why this seems confusing, but this conditional is not designed for this use case. Unfortunately, nothing is currently available for the use case you describe, but we will consider adding something in the future. 🙂

**Comment by kjmartens** [comment]
We will be adding support for this shortly in Freeform 5.x. Just an FYI 🙂 
Should likely be ready later this week or early next week.

**Comment by kjmartens** [comment]
This is now doable in Freeform 5.6+. 🙂 

Use the integration handle for the check: `{% if form.isIntegrationEnabled("integrationHandle") %}`

**Reply by rellafella** [reply]
Thanks @kjmartens I think this is an even better approach. I am sure it will have its benefits for other integrations too.
